crates/sui-core/src/authority_aggregator.rs

```diff
@@ -320,11 +323,23 @@ impl ProcessTransactionState {
             .map(|(digest, (validators, stake))| (*digest, validators, *stake))
     }
 
-    pub fn conflicting_tx_digests_total_stake(&self) -> StakeUnit {
-        self.conflicting_tx_digests
-            .iter()
-            .map(|(_, (_, stake))| *stake)
-            .sum()
+    pub fn record_conflicting_transaction(
```

Comment:
can we merge this with `record_conflicting_transaction_if_any` here and remove one level of indirection as no where else is using this method as is.